indico/modules/users/models/users.py

```diff
@@ -56,6 +56,19 @@ def from_legacy(cls, text):
         return next((x for x in cls if unicode(x.title) == text), None)
 
 
+class NameFormat(TitledIntEnum):
+    __titles__ = (_('John Doe'), _('Doe, John'), _('Doe, J.'), _('J. Doe'),
+                  _('John DOE'), _('DOE, John'), _('DOE, J.'), _('J. DOE'))
+    first_last = 0
```

Comment:
Not a strong opinion, but I would rather have long variables than not clear variables that end up requiring comments.